### PR TITLE
setuptools: update to 53.0.0

### PR DESCRIPTION
**Pull request by naveen521kk**

### CODE REVIEW

File: mingw-w64-python-setuptools/PKGBUILD [hunk 1]
```diff
@@ -58,6 +54,10 @@ prepare() {
   PATH=/mingw64/bin:$PATH gcc -DGUI=0           -O -s -o setuptools-${pkgver}/setuptools/cli-64.exe setuptools-${pkgver}/launcher.c
   PATH=/mingw64/bin:$PATH gcc -DGUI=1 -mwindows -O -s -o setuptools-${pkgver}/setuptools/gui-64.exe setuptools-${pkgver}/launcher.c
 
+  # see https://github.com/pypa/setuptools/commit/9c6e0172e04e260af141ddf0362c6141a375763a
+  cp setuptools-${pkgver}/setuptools/gui-32.exe setuptools-${pkgver}/setuptools/gui.exe
```
Comment (lazka) [inline mingw-w64-python-setuptools/PKGBUILD:58]: What's this about?

Comment (naveen521kk) [reply mingw-w64-python-setuptools/PKGBUILD:58]: I don't know exactly, but that isn't compiled with the patches we have. So, I just copied as specified in https://github.com/pypa/setuptools/commit/9c6e0172e04e260af141ddf0362c6141a375763a

Comment (lazka) [reply mingw-w64-python-setuptools/PKGBUILD:58]: I don't get it. I'd remove this again, I don't see how this can be used.

Comment (naveen521kk) [reply mingw-w64-python-setuptools/PKGBUILD:58]: I think it is used with eggs or something, and yes those are depreciated. will remove these.

File: mingw-w64-python-setuptools/PKGBUILD [hunk 2]
```diff
@@ -8,24 +8,20 @@ provides=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
...
-pkgver=52.0.0
+pkgver=53.0.0
 pkgrel=1
 pkgdesc="Easily download, build, install, upgrade, and uninstall Python packages (mingw-w64)"
 arch=('any')
 license=('PSF')
-url="https://pypi.python.org/pypi/setuptools"
-depends=("${MINGW_PACKAGE_PREFIX}-python>=3.3"
-         "${MINGW_PACKAGE_PREFIX}-python-packaging"
-         "${MINGW_PACKAGE_PREFIX}-python-pyparsing"
```
Comment (lazka) [inline mingw-w64-python-setuptools/PKGBUILD:19]: Why is pyparsing removed?

Comment (naveen521kk) [reply mingw-w64-python-setuptools/PKGBUILD:19]: ah! missed it.

**Pull request merged**